bentoml/archive/archiver.py

```diff
@@ -148,6 +148,8 @@ def _save(bento_service, dst, version=None):
         f.write(BENTO_SERVICE_DOCKERFILE_CPU_TEMPLATE)
     with open(os.path.join(path, 'Dockerfile-sagemaker'), 'w') as f:
         f.write(BENTO_SERVICE_DOCKERFILE_SAGEMAKER_TEMPLATE)
+    with open(os.path.join(path, 'Dockerfile-clipper'), 'w') as f:
+        f.write(BENTO_SERVICE_DOCKERFILE_CLIPPER_TEMPLATE)
```

Comment:
discussed offline, we should handle this similar to DEFAULT_CLIPPER_ENTRY, generate it when calling deploy

Reply:
same for BENTO_SERVICE_DOCKERFILE_SAGEMAKER_TEMPLATE above, but this can be in separate PR

Reply:
done